zubhub_frontend/zubhub/src/assets/js/styles/components/projects_drafts/projectsDraftsGridStyles.js

```diff
@@ -19,18 +19,18 @@ const styles = theme => ({
 
   tabStyle: {
       '&.Mui-selected': {
-        color: '#00B8C4',
+        color: 'var(--primary-color3)',
         fontWeight: 'bold',
         backgroundColor: '#ffffff',

```

Comment:
background color might need to use css variable

Reply:
That is color white, which we haven't asked input from the backend

Reply:
Sure but if the user sets the background color as a light color its hard to read, we might have use the text colora

Reply:
I didn't touch the code where we used plain white, and the fact that we are not allowing to update the background color here, this issue would not rise I guess?